src/storage/src/sink/healthcheck.rs

```diff
@@ -480,17 +480,19 @@ mod tests {
             .unwrap();
 
         let upper = write_handle.upper();
-        let readable_upper = Antichain::from_elem(upper.elements()[0] - 1);
+        let readable_upper = Antichain::from_elem(upper.elements()[0].step_back().unwrap());
 
         read_handle
             .snapshot_and_fetch(readable_upper)
             .await
             .unwrap()
             .into_iter()
             .map(
-                |((v, _), _, _): ((Result<SourceData, String>, Result<(), String>), u64, i64)| {
-                    v.unwrap().0.unwrap()
-                },
+                |((v, _), _, _): (
+                    (Result<SourceData, String>, Result<(), String>),
+                    Timestamp,
+                    i64,
+                )| { v.unwrap().0.unwrap() },
```

Comment:
misc: imagine showing someone unfamiliar with Rust these 5 lines of code and informing them that this is one of the most beloved programming languages